src/styles/main.scss

```diff
@@ -1,3 +1,296 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```

Comment:
checklist p.9 [BEM] - create a separate file per each BEM block styles that have the same name as the block